status-bar/ios/Sources/StatusBarPlugin/UIColor.swift

```diff
@@ -0,0 +1,40 @@
+import Capacitor
+
+public extension CapacitorExtensionTypeWrapper where T: UIColor {
```

Comment:
Could this just be `public extension UIColor`?

Reply:
It could be.
But with just `public extension UIColor`, the method is called by `UIColor.hex(fromColor: backgroundColor)` only instead of `UIColor.capacitor.hex(fromColor: backgroundColor)` like other Capacitor extension methods.

Reply:
Do'h! Yep stick with this.